src/content/docs/en/core-concepts/astro-components.mdx

```diff
@@ -26,20 +22,6 @@ An Astro component is made up of two main parts: the **Component Script** and th
 <!-- Component Template (HTML + JS Expressions) -->
 ```
 
-You can use components inside of other components, to build more and more advanced UI. For example, a `Button` component could be used to create a `ButtonGroup` component like so:
-
-```astro title="src/components/ButtonGroup.astro"
----
-import Button from './Button.astro';
----
-<div>
-  <Button title="Button 1" />
-  <Button title="Button 2" />
-  <Button title="Button 3" />
-</div>
-```
-
-
 ### The Component Script
 
 Astro uses a code fence (`---`) to identify the component script in your Astro component. If you've ever written Markdown before, you may already be familiar with a similar concept called *frontmatter.* Astro's idea of a component script was directly inspired by this concept.
```

Comment:
Whether or not we change the name, I don't think the Markdown frontmatter comparison is helpful enough when they've heard of it to justify the cases when they haven't. They're more visually similar than conceptually similar.